src/elements/content-sidebar/activity-feed/task-form/__tests__/TaskForm-test.js

```diff
@@ -93,6 +93,28 @@ describe('components/ContentSidebar/ActivityFeed/task-form/TaskForm', () => {
         expect(wrapper.find('PillSelectorDropdown').prop('selectorOptions').length).toBe(1);
     });
 
+    test('should not add scrollable class when there are not enough contacts in assignment dropdown', () => {
```

Comment:
we can make good use of [`setProps`](https://airbnb.io/enzyme/docs/api/ShallowWrapper/setProps.html) here.